### DISCUSSION TITLE
Twitter Space: Getting Started with Accessibility

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#44] [News + Announcements]
Join [Bekah](https://twitter.com/BekahHW) and [Chris DeMars](https://twitter.com/saltnburnem) for a Twitter Space on Getting Started with Accessibility! 

When: Wednesday, Nov 2, 10a PT | 1p ET
Where: [@DeepgramAI](https://twitter.com/DeepgramAI) on Twitter

![image card with details of event and headshot of the speakers](https://user-images.githubusercontent.com/34313413/195156236-559defa7-9cf0-4bfc-b477-eb7e3e9a8cce.png)

If you have questions you'd like to see us address, drop it in the replies.

**Comment by ielrekaa** [comment]
How do you define the need for accessibility? 
Is it based on WCAG guidelines? Google SEO? Disabilities that are not as easy to measure the impact for such as like dyslexia, ADHD, red-green colorblindness, autism etc.?

**Comment by ielrekaa** [comment]
In addition to plain transcription, how can ASR improve accessibility? How can it help different groups of people? Or different situations anyone can be in?
